src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-default&quot;&gt;Secondary&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-link">Link</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              Link buttons trigger actions that take users to other pages or actions that end tasks or exit modals (such as <strong>Cancel</strong>).
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-link&quot;&gt;Link button&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-link-inline">Inline link</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              Inline link buttons trigger the same type of actions as link buttons, but the styling is altered to fit inline.
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-link-inline&quot;&gt;Inline link&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-primary sky-btn-disabled">Primary</button>
+            </div>
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-default sky-btn-disabled">Secondary</button>
+            </div>
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-link sky-btn-disabled">Link</button>
+            </div>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              The disable class disables all button types. The most common use case is to disable actions until users complete prerequisite tasks.
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>class="sky-btn-disabled"</sky-code>
```

Comment:
I know we're just copy/pasting from what was there before, but why do all of our code snippets include `<button ... >` while this last one does not? Should they all be consistent?

Reply:
I assume this last one doesn't include the `button` element because it would require a lot of real estate to include the full `button` code snippet for all four of the buttons in the lefthand column. It is definitely inconsistent, but I'm not sure that it's really worth the real estate to list all 4 complete code snippets. What do you think, @Blackbaud-AlexKingman?

Reply:
What if we just whittled each one down to the `class` information? For example `class="sky-btn sky-btn-primary"`
We don't need to tell consumers how to write button HTML, do we?

Reply:
I'll defer to you Alex on whether or not we need to include the button HTML. I will say though that removing it is much easier on the eye.

Reply:
I agree with Denise that it looks better, but I do think there's some value in showing the full button HTML. I can't speak for all consumers, but when I've created buttons in the past, it was helpful for me. I agree with Denise and defer to you Alex on whether we really need the `button` HTML around the classes, but I do think it's worth factoring in the fact that these docs don't have a code sample for users to see the classes in place.

Reply:
You can also apply these classes to an `<a>` tag, so I'm not really sure that showing button markup is adding a ton of value. Just seems like extra noise to me. I'd pare it down to just the class if it were up to me. If we get complaints, then maybe a demo would be appropriate here.

Reply:
I went ahead and removed this for now. We can add it back if we hear otherwise